src/app/features/home/home.component.ts

```diff
@@ -1,10 +1,20 @@
 import { Component } from '@angular/core';
+import { HomeService } from 'src/app/core/home.service';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
-  styleUrls: ['./home.component.css']
+  styleUrls: ['./home.component.css'],
 })
 export class HomeComponent {
-
+  private service = new HomeService();
```

Comment:
Please read t[his link about services and dependency injection](https://angular.io/guide/architecture-services) and adjust your code accordingly.